package.json

```diff
@@ -33,5 +40,9 @@
     "ts-jest": "^28.0.4",
     "ts-node": "^10.8.0",
     "typescript": "^4.6.4"
+  },
+  "resolutions": {
+    "@segment/analytics-next": "workspace:*",
```

Comment:
just for my curiosity ... what does this do?

Reply:
I stole this pattern in another repo, but I don't remember which one.

It should allow us to have fixed version numbers in our packages for the sake of publishing, but when developing locally or running tests, this resolutions key will always guarantee that both our dependencies and any transient dependencies will use the "current" workspace, rather than (accidentally?) using the version that is published (or maybe not-yet published) to npm.  For example, if we have a not-yet-published `@segment/utils` that is updated, and is a transient dependency of `analytics-core` that is installed in `analytics-node`. Will an install pick up on the local changes when we run our tests? I could be wrong, but I don't _think_ so.

Reply:
You can see an example here: https://github.com/facebook/jest/blob/main/package.json#L172